csharp/ql/test/library-tests/assignments/AssignOperationExpanded.ql

```diff
@@ -2,18 +2,19 @@ import csharp
 
 predicate getExpandedOperatorArgs(Expr e, Expr left, Expr right) {
   e = any(BinaryArithmeticOperation bo |
-    bo.getLeftOperand() = left and
-    bo.getRightOperand() = right
-  )
+      bo.getLeftOperand() = left and
+      bo.getRightOperand() = right
+    )
```

Comment:
@aschackmull : Is the indentation here as expected?

Reply:
Yes. There's one level of indentation induced by the equality and one level induced by the `any`.